pkg/cmd/controller/pipelinerunner.go

```diff
@@ -144,6 +145,51 @@ func (o *PipelineRunnerOptions) Run() error {
 	return nil
 }
 
+func (o *PipelineRunnerOptions) configure() error {
+	err := o.setLogLevel()
+	if err != nil {
+		return errors.Wrap(err, "unable to set log level")
+	}
+
+	logger.Infof("meta pipeline enabled: %t", o.UseMetaPipeline)
+	if o.UseMetaPipeline {
+		logger.Info("creating meta pipeline client")
+		client, err := metapipeline.NewMetaPipelineClient()
+		if err != nil {
+			return errors.Wrap(err, "unable to create meta pipeline client")
+		}
+
+		client.SetServiceAccount(o.ServiceAccount)
+		client.SetDefaultImage(o.MetaPipelineImage)
+
+		o.metaPipelineClient = client
+	} else {
+		if !o.NoGitCredentialsInit {
+			err := o.InitGitConfigAndUser()
+			if err != nil {
+				return err
+			}
+
+			err = o.stepGitCredentials()
+			if err != nil {
+				logger.Warn(err.Error())
```

Comment:
Shouldn't this fail here instead of just logging the error and continue?

Reply:
+1